src/app/modules/webhook/webhook.middleware.ts

```diff
@@ -0,0 +1,44 @@
+import { Request, RequestHandler } from 'express'
+
+import { createLoggerWithLabel } from 'src/config/logger'
```

Comment:
e2e tests are failing because contrary to popular belief, `baseUrl` somehow does not work (probably because of misconfigured webpack) properly when transpiled :P.

use relative paths instead